relayer/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist",
+    "allowJs": false,
+    "target": "es5",
+    "noImplicitAny": false,
+    "skipLibCheck": true
+  },
+  "include": ["./src/**/*"]
+}
```

Comment:
I would really-really like to see this set to `noImplicitAny: true` alongside with strict mode enabled. Also since it is a backend code for modern Node.js, `target: es5` seems too concervative.